source/install/desktop-app-install.rst

```diff
@@ -191,32 +191,32 @@ To resolve this issue, you can reset the desktop app screen location by deleting
 
 **To reset the desktop app screen location**
 
-1. If the desktop app is running, right-click the Mattermost icon in the task bar and click **Close Window**.
-2. Open Windows File Explorer, and navigate to the ``%APPDATA%\\Mattermost`` folder.
+1. If the desktop app is running, right-click the Mattermost icon in the task bar, then select **Close Window**.
+2. Open Windows File Explorer, and go to the ``%APPDATA%\\Mattermost`` folder.
 3. Delete the file ``bounds-info.json``.
 
 Desktop App constantly refreshes the page
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 This issue can occur when ``localStorage`` has an unexpected state. To resolve the issue:
 
-- Windows: Open Windows File Explorer, navigate to the ``%APPDATA%\Mattermost`` folder, then delete the ``Local Storage`` folder.
-- Mac: Open Finder, navigate to the ``~/Library/Application Support/Mattermost`` folder, then delete the ``Local Storage`` folder.
-- Linux: Open the File Manager, navigate to the ``~/.config/Mattermost`` folder, then delete the ``Local Storage`` folder.
+- Windows: Open Windows File Explorer, go to the ``%APPDATA%\Mattermost`` folder, then delete the ``Local Storage`` folder.
+- Mac: Open Finder, go to the ``~/Library/Application Support/Mattermost`` folder, then delete the ``Local Storage`` folder.
+- Linux: Open the File Manager, go to the ``~/.config/Mattermost`` folder, then delete the ``Local Storage`` folder.
```

Comment:
Linux filemanagers hide folders starting with a period by default. I'd recommend deleting them using the terminal using `rm -rf ~/.config/Mattermost` or similar.